package/com.unity.formats.usd/Samples/ExportMeshWithAnimation/Explode.cs

```diff
@@ -18,33 +18,25 @@ namespace USD.NET.Examples
 {
     public class Explode : MonoBehaviour
     {
-        public int m_explodeTime = 10;
+        public float m_explodeTime = 10;
         public Transform m_effectRoot;
         public float m_force = 1;
         public float m_radius = 1;
 
-        private bool m_active = true;
-
-        void Start()
-        {
-        }
+        private float currTime;
 
         void Update()
         {
-            if (!m_active)
-            {
-                return;
-            }
+            currTime += Time.deltaTime;
```

Comment:
yes its to do the explode animation repeatedly - i made the change so that if the User wants to record the explode animation but misses the explode animation they dont have to restart the scene